<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>get</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>post</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>put</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>delete</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>线性回归</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>xgboost</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LSTM</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>get</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>post</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>put</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>delete</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
